### ISSUE TITLE
Update Afrikaans translations

### ISSUE THREAD

**Issue by spac3** [squidfunk/mkdocs-material#5883]
### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "af",
  "action.edit": "Wysig hierdie bladsy",
  "action.skip": "Slaan oor na inhoud",
  "action.view": "Bekyk bron van hierdie bladsy ⬅️",
  "announce.dismiss": "Moenie dit weer wys nie ⬅️",
  "blog.archive": "Argief ⬅️",
  "blog.categories": "Kategorieë ⬅️",
  "blog.categories.in": "binne ⬅️",
  "blog.continue": "Lees verder ⬅️",
  "blog.draft": "Konsep ⬅️",
  "blog.index": "Terug na indeks ⬅️",
  "blog.meta": "Metadata ⬅️",
  "blog.references": "Verwante skakels ⬅️",
  "clipboard.copy": "Kopieer na knipbord",
  "clipboard.copied": "gekopieer na knipbord",
  "consent.accept": "Aanvaar ⬅️",
  "consent.manage": "Bestuur instellings ⬅️",
  "consent.reject": "Verwerp ⬅️",
  "footer": "Voetskrif ⬅️",
  "footer.next": "Volgende",
  "footer.previous": "Vorige",
  "header": "Kopskrif ⬅️",
  "meta.comments": "Kommentaar",
  "meta.source": "Bron",
  "nav": "Navigasie ⬅️",
  "readtime.one": "1 minuut se lees ⬅️",
  "readtime.other": "# minuut se lees ⬅️",
  "rss.created": "RSS-voer geskep ⬅️",
  "rss.updated": "RSS-voer van opgedateerde inhoud ⬅️",
  "search": "Soek ⬅️",
  "search.placeholder": "Soek",
  "search.share": "Deel ⬅️",
  "search.reset": "Terugstel ⬅️",
  "search.result.initializer": "Inisialisering van soektog ⬅️",
  "search.result.placeholder": "Tik om te begin soek",
  "search.result.none": "Geen ooreenstemmende dokumente",
  "search.result.one": "1 ooreenstemmende dokument",
  "search.result.other": "# ooreenstemmende dokumente",
  "search.result.more.one": "1 meer op hierdie bladsy ⬅️",
  "search.result.more.other": "# meer op hierdie bladsy ⬅️",
  "search.result.term.missing": "Vermis ⬅️",
  "select.language": "Kies taal ⬅️",
  "select.version": "Kies weergawe ⬅️",
  "source": "Slaan oor na inhoud",
  "source.file.contributors": "Medewerkers ⬅️",
  "source.file.date.created": "Geskep",
  "source.file.date.updated": "Laaste opdatering",
  "tabs": "Duimgids ⬅️",
  "toc": "Inhoudsopgawe",
  "top": "Terug na bo ⬅️"
}[key] }}{% endmacro %}
```

### Before submitting

- [X] I've checked the list of [available languages](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language) for existing translations.
- [X] I assure that the translations are accurate to the best of my knowledge.
- [ ] __Optional__: I want to integrate this translation myself and create a pull request following the [contribution guide](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md).

**Comment by squidfunk** [comment]
Awesome, thanks! Added in d9456543c.

**Comment by squidfunk** [comment]
Released as part of [9.2.4](https://squidfunk.github.io/mkdocs-material/changelog/#9.2.4).
